### PR TITLE
Include weeks as possible duration unit

### PR DESCRIPTION
**Pull request by carolksun**
This PR adds weeks as a possible duration unit, which was previously missed in
PR#39.

### CODE REVIEW

**Review by LujieZ — DISMISSED** [review]

File: web/src/app/components/duration.utils.ts [hunk 1]
```diff
@@ -21,32 +21,31 @@ export interface DurationItem {
...
 export function findDurationUnit(pushInfos: step189_2020.IPushInfo[]): string {
-  const unitCounter: {[unit: string]: number} = {
-    seconds: 0,
-    minutes: 0,
-    hours: 0,
-    days: 0
-  };
+  const unitCounter:
+      {[unit: string]:
+           number} = {seconds: 0, minutes: 0, hours: 0, days: 0, weeks: 0};
```
Comment (yinggao) [inline web/src/app/components/duration.utils.ts:48]: Can we keep the original formatting?

Comment (carolksun) [reply web/src/app/components/duration.utils.ts:48]: the clang formatter doesn't let me but Razvan mentioned it worked for him?

Comment (razvanm) [reply web/src/app/components/duration.utils.ts:48]: The formatter seem use the long format if there is a `,` at the end. 

Here is a form that works:

```
  const unitCounter: {[unit: string]: number} = {
    seconds: 0,
    minutes: 0,
    hours: 0,
    days: 0,
    weeks: 0,
  };
```

**Review by yinggao — DISMISSED** [review]

File: web/src/app/components/duration.utils.ts [hunk 2]
```diff
@@ -21,31 +21,34 @@ export interface DurationItem {
...
+const NANO_TO_DAYS = (10 ** 9) * 60 * 60 * 24;
+const NANO_TO_WEEKS = (10 ** 9) * 60 * 60 * 24 * 7;
 
 export const UNIT_CONVERSION: {[unit: string]: number} = {
   seconds: NANO_TO_SECONDS,
   minutes: NANO_TO_MINUTES,
   hours: NANO_TO_HOURS,
-  days: NANO_TO_DAYS
+  days: NANO_TO_DAYS,
+  weeks: NANO_TO_WEEKS
```
Comment (razvanm) [inline web/src/app/components/duration.utils.ts:35]:
```suggestion
  weeks: NANO_TO_WEEKS,
```

**Pull request merged**